stage-1/set-python-package-version/action.yml

```
@@ -53,21 +53,29 @@ runs:
           git add "$INIT_FILE"
         fi
 
-    - name: Get setup.py version
+    - name: Get package version
       # Might differ slightly from __init__.py, e.g. if epoch prefixed
-      id: get-setup-py-version
+      id: get-package-version
       shell: bash
       run: |
-        setup_py_version=$( python setup.py --version )
+        if [[ ! -f setup.py ]]; then
```

I think this is why the tests were failing
```suggestion
        if [[ -f setup.py ]]; then
```